AutomationHelpers/src/UserCodeCollections/DragNDropLibrary.cs

```diff
@@ -0,0 +1,46 @@
+﻿/*
+ * Created by Ranorex
+ * User: sknopper
+ * Date: 9/19/2017
+ * Time: 12:22 PM
+ * 
+ * To change this template use Tools | Options | Coding | Edit Standard Headers.
+ */
+using System.Drawing;
+
+using Ranorex;
+
+using Ranorex.Core.Testing;
+
+namespace CheckAutomationHelperIntegration.UserCodeCollections
+{
+    /// <summary>
+    /// Ranorex User Code collection. A collection is used to publish User Code methods to the User Code library.
+    /// </summary>
+    [UserCodeCollection]
+    public class DragNDropLibrary
+    {
+    	
+    	/// <summary>
+    	/// This is a placeholder text. Please describe the purpose of the
```

Comment:
Missing summary, this is still the original placeholder text. Please provide a description of the method

Reply:
Updated, sorry for that.